compiler/runtime.go

```diff
@@ -572,6 +573,23 @@ type functionValue struct {
 	scope  Scope
 }
 
+// functionValue implements the interpreter.Resolver interface.
+var _ interpreter.Resolver = (*functionValue)(nil)
```

Comment:
Is the only purpose of this line of code to document that `functionValue` implements `interpreter.Resolver`? If so, that's awesome.

Reply:
Yes, I've seen it elsewhere in our code. This way if somehow this struct stops implementing the interface, the error will occur close to where the fix needs to happen.

Reply:
That is so useful. I'm definitely going to start doing that.